tests/test_frequency_calculator.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from pathlib import Path
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+from typer.testing import CliRunner
+
+from frequency_calculator import app
+
+runner = CliRunner()
```

Comment:
Fixture

Reply:
Okey... at first I couldn't undestand why a fixture.
But then it hit me. It is connected with creating CliRunner every time in every test file. And a fixture solves it. 
Because I have only one test file with CliRunner and don't have a big projects where it was needed several runners yet, I needed time to undestand it =) 
So I tried this and rewrote my test file with adding a fixture.